### PR TITLE
feat: docker compose (#89)

### LINKED ISSUE #89: docker compose file with Stargate for local dev

### PR DESCRIPTION
**Pull request by eddiejaoude**
closes #89 

- running Stargate locally with docker
-

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
One Action is running for about 45 min now 😅

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: '3.9'
+services:
+    stargate:
+        image: 'stargateio/stargate-3_11:v1.0.0'
```
Comment (Cahllagerfeld) [inline docker-compose.yml:4]: In the [docs](https://stargate.io/docs/stargate/1.0/quickstart/quick_start-document.html) they use `stargateio/stargate-3_11:v1.0.13`
Do you think this matters?

Comment (eddiejaoude) [reply docker-compose.yml:4]: ah good spot! 👍 looks like they have bumped the build versions, probably more important when they move minor versions

**Comment by eddiejaoude** [comment]
> One Action is running for about 45 min now 😅

not sure why that is 🤔

**Review by Cahllagerfeld — APPROVED** [review]

**Pull request merged**